mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: "GeoJSON PHP library"
 repo_url: https://github.com/jmikola/geojson
-theme: spacelab
-pages:
-    - [index.md, Home]
+theme: readthedocs
+nav:
+    - Home: index.md
```

Comment:
If the API documentation is being removed I'm not sure it makes much sense to continue using `mkdocs` for a single file that could be incorporated into the README. That said, I can always take care of that after this PR.

Reply:
If you'd like, I can do it in a subsequent PR (I would like to do some more cleanups and quality of life improvements in other PRs, if you have the time and headspace to review them 🤞)